software/nosql/pom.xml

```diff
@@ -295,4 +295,26 @@
         </plugins>
       </pluginManagement>
     </build>
+
+  <profiles>
+    <profile>
+      <id>defaultProfile</id>
+      <activation>
+	<property><name>!brooklyn.cloudtest</name></property>
+      </activation>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <configuration>
+              <excludes>
+                <exclude>**/CassandraFabricTest.java</exclude>
```

Comment:
Is the test failing? If that's the case, then better to exclude it in the release branch only.